src/voronoi_visual_utils.hpp

```diff
@@ -0,0 +1,187 @@
+
+// Boost.Polygon library voronoi_graphic_utils.hpp header file
+
+//          Copyright Andrii Sydorchuk 2010-2012.
+// Distributed under the Boost Software License, Version 1.0.
+//    (See accompanying file LICENSE_1_0.txt or copy at
+//          http://www.boost.org/LICENSE_1_0.txt)
+
```

Comment:
We should check if the boost library doesn't bite with AGPL

Reply:
According to GNU, Boost is compatible: https://www.gnu.org/licenses/license-list.en.html#boost